src/OmniSharp.Abstractions/Models/v2/BlockSpan.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace OmniSharp.Models.V2
+{
+    public class BlockSpan
+    {
+        public BlockSpan(Range textSpan, Range hintSpan, string bannerText,  string type)
+        {
+            TextSpan = textSpan;
+            HintSpan = hintSpan;
+            BannerText = bannerText;
+            Type = type;
+        }
+
+        /// <summary>
+        /// The span of text to collapse.
+        /// </summary>
+        public Range TextSpan { get; }
```

Comment:
We might consider using "Range" terminology here. I don't think "Span" appears anywhere in OmniSharp's protocol.